### PR TITLE
Compatibility Updates for the Refactoring Branch

### PR DESCRIPTION
**Pull request by nerisaurus**
After some more testing, I believe this branch handles the latest updates to Proteus (with regards to Shape and Boundary Condition API changes) for the numericalTanks problems.  All are able to run and are making waves which look right to me.  The code has several spots I want to return to later (like imports), but for now this should be good in keeping them functional.

There are also some changes to error outputting and some minor formatting changes (but as noted before, the main pass for that kind of nitpicky refactoring will come later, after more drastically obsolete tests are brought back to speed).

### CODE REVIEW

File: 2d/numericalTanks/linearWaves/tank2D.py [hunk 1]
```diff
@@ -200,7 +193,7 @@
     rd_shockCapturingFactor  = 0.5
     rd_lag_shockCapturing = False
     epsFact_density    = 3.0
-    epsFact_viscosity  = epsFact_curvature  = epsFact_vof = epsFact_consrv_heaviside = epsFact_consrv_dirac = epsFact_density
+    epsFact_viscosity  = epsFact_curvature  = epsFact_vof = ecH = epsFact_consrv_dirac = epsFact_density
```
Comment (nerisaurus) [inline 2d/numericalTanks/linearWaves/tank2D.py]: I know there were discussions about what to do with WaveTools and other things needing a variable named `ecH`.  What are the plans in that regard?  If we want to stick (for style or convenience) with `ecH`, how should this naming convention work with the other variables here (which similarly pop up in a lot of the air-water-vv set)?

**Comment by nerisaurus** [comment]
I'm aiming to merge this soon in order to make a clean start on a new refactoring pass for the numerical tanks after what I've learned in refactoring the benchmark cases.  If anyone notices issues with these updates, you should note your concerns.
